util/sqlexec/utils_test.go

```diff
@@ -102,14 +96,16 @@ func (s *testUtilsSuite) TestEscapeBackslash(c *C) {
 			output: []byte("中文?"),
 		},
 	}
-	for _, t := range tests {
-		commentf := Commentf("%s", t.name)
-		c.Assert(escapeBytesBackslash(nil, t.input), DeepEquals, t.output, commentf)
-		c.Assert(escapeStringBackslash(nil, string(t.input)), DeepEquals, t.output, commentf)
+	for _, v := range tests {
+		t.Run(v.name, func(t *testing.T) {
+			require.Equal(t, v.output, escapeBytesBackslash(nil, v.input))
```

Comment:
Loops should be able to paralleled, i.e. move the `Parallel()` inside the loop.

Reply:
you mean add `Parallel()` to the anonymous function of t.Run, right?

Reply:
fixed as my understanding